apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -99,31 +106,30 @@ export default function UserThreadPanel({
     setLoading(true)
     let moreThreads: Thread[] = []
     const userOnSlug = userWithSlug.value?.user
-    const isOwnProfile = session?.user?.id === userOnSlug?.userId;
-    if (page === 'profile' && !session?.user || !isOwnProfile) {
-      moreThreads = await fetchBrowseThreads();
+    const isOwnProfile = session?.user?.id === userOnSlug?.userId
+    if ((page === 'profile' && !session?.user) || !isOwnProfile) {
+      moreThreads = await fetchBrowseThreads()
     } else {
       moreThreads = await getThreads({
         jwt: session!.user?.hasuraJwt,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)